src/GlobalStyle.js

```diff
@@ -31,8 +32,9 @@ export default createGlobalStyle`
    *::before,
    *::after {
      box-sizing: border-box;
-     background: var(--primary);
+     //ackground: var(--primary);
```

Comment:
do you actually need the background property? it seems like you don't, so you can delete the comment as well :)

Reply:
Thank you for the hint! I left some CSS tryouts as comments. I'll come back to them and delete them.